packages/pds/src/app-view/api/app/bsky/unspecced.ts

```diff
@@ -192,6 +192,43 @@ export default function (server: Server, ctx: AppContext) {
     },
   })
 
+  server.app.bsky.unspecced.registerPushNotification({
+    auth: ctx.accessVerifier,
+    handler: async ({ auth, params }) => {
+      const { token, platform, appId, endpoint } = params
+      const {
+        credentials: { did },
+      } = auth
+
+      const { appviewAgent } = ctx
+
+      if (ctx.canProxyWrite()) {
+        // TODO: hit the endpoint here so that 3rd party devs can run their own notification servers
+        // const headers = await ctx.serviceAuthHeaders(did)
```

Comment:
mind cleaning up this comment - leaving the todo is great 👌 

(altho such a nit but can we write it as `@TODO`)